### PR TITLE
Add click command for sending email to unconsented samples

### PR DESCRIPTION
**Pull request by squirrelo**
does what it says on the tin.

### CODE REVIEW

**Comment by EmbrietteH** [comment]
I'm ok with the text as it is.
ping @mortonjt @JWDebelius @ackermag for extra set of eyes on code and text please!

File: knimin/lib/data_access.py [hunk 1]
```diff
@@ -1070,6 +1070,22 @@ def authenticate_user(self, email, password):
 
         return False
 
+    def get_unconsented(self):
+        """Returns unconsented barcode and person's email
+
+        Returns
+        -------
+        list of (str, datetime.date, str)
+            Unconsted barcodes, in the form [(barcode, scan_date, email), ...]
```
Comment (mortonjt) [inline knimin/lib/data_access.py]: Unconsted -> Unconsented

File: knimin/lib/tests/test_geocoder.py [hunk 2]
```diff
@@ -76,12 +76,12 @@ def test_geocode_nonmock(self):
         obs = geocode('9500 Gilman Dr, La Jolla, CA')
         exp = Location('9500 Gilman Dr, La Jolla, CA', 32.8794239,
                        -117.2369135, 114.7895736694336, 'San Diego', 'CA',
-                       '92093', 'United States')
+                       '92037', 'United States')
```
Comment (mortonjt) [inline knimin/lib/tests/test_geocoder.py]: For future reference, its good practice to add in your commit message why these values changed.

**Comment by mortonjt** [comment]
:+1:

**Comment by squirrelo** [comment]
Ping @mortonjt @EmbrietteH can this get merged before more conflicts arise?

**Pull request merged**